packages/remix-dev/__tests__/readConfig-test.ts

```diff
@@ -65,15 +65,11 @@ describe("readConfig", () => {
           },
         },
         "serverBuildPath": Any<String>,
-        "serverBuildTarget": undefined,
         "serverBuildTargetEntryModule": "export * from \\"@remix-run/dev/server-build\\";",
         "serverConditions": undefined,
         "serverDependenciesToBundle": Array [],
         "serverEntryPoint": undefined,
-        "serverMainFields": Array [
-          "main",
-          "module",
-        ],
```

Comment:
This should be kept, otherwise we're not doing the same thing anymore when compiling